usr/lib/ep11_stdll/ep11_specific.c

```diff
@@ -434,7 +451,17 @@ typedef struct {
     short apqns[2 * MAX_APQN];
 } __attribute__ ((packed)) ep11_target_t;
 
+static CK_RV handle_all_ep11_cards(ep11_target_t * ep11_targets,
+                                   adapter_handler_t handler,
+                                   void *handler_data);
+
 /* EP11 token private data */
+#define MK_VP_LENGTH                32
```

Comment:
HM, MK is ambiguous. There are crypto card MKs and LPAR/Virtual Guest MKs. How about naming this PKEY_MK_VP_LENTH?

Reply:
Done.